src/BuildingRegistry.Api.BackOffice/Building/BuildingController-Demolish.cs

```diff
@@ -0,0 +1,79 @@
+namespace BuildingRegistry.Api.BackOffice.Building
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Abstractions.Building.Requests;
+    using Abstractions.Building.Validators;
+    using Abstractions.Building.SqsRequests;
+    using Abstractions.Validation;
+    using Be.Vlaanderen.Basisregisters.Auth.AcmIdm;
+    using Be.Vlaanderen.Basisregisters.Api.ETag;
+    using Be.Vlaanderen.Basisregisters.Api.Exceptions;
+    using Be.Vlaanderen.Basisregisters.GrAr.Provenance;
+    using BuildingRegistry.Building;
+    using Infrastructure;
+    using Microsoft.AspNetCore.Authentication.JwtBearer;
+    using Microsoft.AspNetCore.Authorization;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using NodaTime;
+    using Swashbuckle.AspNetCore.Filters;
+
+    public partial class BuildingController
+    {
+        /// <summary>
+        /// Gebouw slopen.
+        /// </summary>
+        /// <param name="buildingExistsValidator"></param>
+        /// <param name="ifMatchHeaderValidator"></param>
+        /// <param name="request"></param>
+        /// <param name="ifMatchHeaderValue"></param>
+        /// <param name="cancellationToken"></param>
+        [HttpPost("{persistentLocalId}/acties/slopen")]
+        [ProducesResponseType(StatusCodes.Status202Accepted)]
+        [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status500InternalServerError)]
+        [SwaggerResponseHeader(StatusCodes.Status202Accepted, "location", "string", "De url van het gebouw.")]
+        [SwaggerResponseExample(StatusCodes.Status400BadRequest, typeof(BadRequestResponseExamples))]
+        [SwaggerResponseExample(StatusCodes.Status500InternalServerError, typeof(InternalServerErrorResponseExamples))]
+        [Authorize(AuthenticationSchemes = JwtBearerDefaults.AuthenticationScheme, Policy = PolicyNames.GeschetstGebouw.DecentraleBijwerker)]
+        public async Task<IActionResult> Demolish(
+            [FromServices] BuildingExistsValidator buildingExistsValidator,
+            [FromServices] IIfMatchHeaderValidator ifMatchHeaderValidator,
```

Comment:
We don't do if match header validation as request are done by ourselves based upon lines in the uploaded grb file.